operator/github/v0/README.mdx

```diff
@@ -0,0 +1,266 @@
+
+
+
+
+
+### Get Commit
+
+Get a commit from a repository, given the commit SHA
+
+
+| Input | ID | Type | Description |
+| :--- | :--- | :--- | :--- |
+| Task ID (required) | `task` | string | `TASK_GET_COMMIT` |
+| Owner (required) | `owner` | string | owner of the repository |
+| Repository (required) | `repository` | string | repository name |
+| Commit SHA (required) | `sha` | string | sha of the commit |
+
+
+
+| Output | ID | Type | Description |
+| :--- | :--- | :--- | :--- |
+| Commit | `commit` | object | A commit in GitHub |
+
+
+
+
+
+
+### Get Review Comments
+
+Get the review comments in a pull request
+
+
+| Input | ID | Type | Description |
+| :--- | :--- | :--- | :--- |
+| Task ID (required) | `task` | string | `TASK_GET_REVIEW_COMMENTS` |
+| Owner (required) | `owner` | string | owner of the repository |
+| Repository (required) | `repository` | string | repository name |
+| PR Number | `pr_number` | integer | Number of the PR. Default is the latest PR |
+| Sort | `sort` | string | Sort the comments by created, updated. Default: created |
+| Direction | `direction` | string | Direction of the sort, including asc or desc. Default: asc |
+| Since | `since` | string | Only comments updated at or after this time are returned. Default: 2021-01-01T00:00:00Z |
+
+
+
+| Output | ID | Type | Description |
+| :--- | :--- | :--- | :--- |
+| Comments | `comments` | array[object] | An array of comments |
+
+
+
+
+
+
+### Create Review Comment
+
+Create a review comment in pull request.
+
+
+| Input | ID | Type | Description |
+| :--- | :--- | :--- | :--- |
+| Task ID (required) | `task` | string | `TASK_CREATE_REVIEW_COMMENT` |
+| Owner (required) | `owner` | string | owner of the repository |
+| Repository (required) | `repository` | string | repository name |
+| PR Number (required) | `pr_number` | integer | Number of the PR. Default is the latest PR |
+| Comment (required) | `comment` | object | The comment to be added |
+
+
+
+| Output | ID | Type | Description |
+| :--- | :--- | :--- | :--- |
+| Review Comment | `comment` | object | The created comment |
+
+
+
+
+
+
+### Get All Issues
+
+Get the list of all issues in a repository
+
+
+| Input | ID | Type | Description |
+| :--- | :--- | :--- | :--- |
+| Task ID (required) | `task` | string | `TASK_GET_ALL_ISSUES` |
+| Owner (required) | `owner` | string | owner of the repository |
+| Repository (required) | `repository` | string | repository name |
+| State | `state` | string | State of the issues, including open, closed, all. Default: open |
+| Sort | `sort` | string | Sort the issues by created, updated, popularity, or long-running. Default: created |
+| Direction | `direction` | string | Direction of the sort, including asc or desc. Default: asc |
+| Since | `since` | string | Only issues updated at or after this time are returned. Default: 2021-01-01T00:00:00Z |
+| No Pull Request | `no_pull_request` | boolean | Whether to not include pull requests in the issues. Default: false |
```

Comment:
```suggestion
| State | `state` | string | State of the issues, including open, closed, all. Default is open |
| Sort | `sort` | string | Sort the issues by created, updated, popularity, or long-running. Default is created |
| Direction | `direction` | string | Direction of the sort, including asc or desc. Default is asc |
| Since | `since` | string | Only issues updated at or after this time are returned. Default is 2021-01-01T00:00:00Z |
| No Pull Request | `no_pull_request` | boolean | Whether to not include pull requests in the issues. Default is false |
```